lib/redix/connection.ex

```diff
@@ -144,14 +144,14 @@ defmodule Redix.Connection do
       "Disconnected from Redis (", Utils.format_host(state), "): ", Exception.message(error),
     ])
 
-    :ok = :gen_tcp.close(state.socket)
+    :ok = :ssl.close(state.socket)
```

Comment:
does this only support ssl now and if so how does that effect our dev environments?

Reply:
This will make it only support ssl connections. Rafe said it's fine. Otherwise we'd be spending a lot of time figuring out how to make it work with either